frontend/src/components/AdminSuspendModal/AdminSuspendModal.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {
+  Button, Modal, ModalHeader, ModalBody, ModalFooter,
+} from 'reactstrap';
+import './AdminSuspendModal.css';
+import { suspendUser } from '../../lib/apiClient';
+
+class AdminSuspendModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+    };
+  }
+
+  render() {
+    const {
+      toggleSuspendModal, suspendModal, selectedUser, refreshUsers,
+    } = this.props;
+
+    const handleSuspend = () => {
+      suspendUser(selectedUser, selectedUser.id).then(() => {
+        toggleSuspendModal();
+        refreshUsers();
+      });
+    };
+
+    const closeBtn = <button type="button" className="suspend-user-close-button" onClick={toggleSuspendModal}><img src="https://img.icons8.com/windows/32/9b9b9b/cancel.png" alt="close" /></button>;
```

Comment:
I'd also break this into multiple lines, and any text below that causes a linting issue.